### ISSUE TITLE
Listbox scrollbar

### ISSUE THREAD

**Issue by Pachy** [lawsie/guizero#275]
When a Listbox.resize() is called, the scrollbar disappears.  Also happens if you specify width and height on construction.

**Comment by martinohanlon** [comment]
Sounds like a bug that does :)

Do you have an example program I can use to reproduce the error it'll save me some time.

**Comment by Pachy** [comment]
from guizero import App, Text, PushButton, Box, Window, ListBox
app = App(title="piCD", height=250)
SelList = ListBox(app, grid=[0,1], align="left", scrollbar=True)
for x in range(20):
	SelList.append(str(x))
#SelList.resize(450, 200)		# DELETES THE SCROLL BAR
app.display()

**Comment by martinohanlon** [comment]
I think this issue is related to #272 - the scrollbar is a tkinter widget packed into a guizero container and when its resized it all goes wrong. 

When #272 is resolved, ListBox can be refactored to use this functionality to insert the tk scrollbar into itself.

When I created ListBox I put a note at the top stating:

```
# I suspect the implementation of ListBox can be simplified in the future...
# At the moment it is a Container which contains a list box
# The only reason it is a container is to support scrollbars which need to be placed inside a frame.
# It feels like scrollbar should be built into guizero's base classes, Im just not sure how at the moment.
# Martin
```

The time has come :)

**Comment by martinohanlon** [comment]
@Pachy I have got to the bottom of the issue. The `ListBox` object is a `frame` which contains a tkinter `listbox` and a `scrollbar`, when the `ListBox` `frame` is resized it also sets the `listbox` to be the same size, unfortunately this means the `scrollbar` is "covered" up and hidden.

I am going to do some refactoring of `ListBox` but if you want to get your solution working you can resize the internal `listbox` after you resize the `ListBox` like this:

```python
from guizero import App, Text, PushButton, Box, Window, ListBox
app = App(title="piCD", height=250)
SelList = ListBox(app, grid=[0,1], align="left", scrollbar=True)
for x in range(20):
    SelList.append(str(x))
SelList.resize(500, 200)
# Reset the internal listbox size
SelList._listbox.resize(None, None)
# Set the listbox size to fill the available space
SelList._listbox.resize("fill", "fill")
app.display()
```

You will need to be using the latest version of guizero.

**Comment by martinohanlon** [comment]
Fixed in `dev`. It will be in the next release.

**Comment by martinohanlon** [comment]
Released in `0.6.2`
